addons/killtracker/XEH_postInit.sqf

```diff
@@ -43,15 +43,15 @@ GVAR(killCount) = 0;
     TRACE_2("kill eh",_name,_killInfo);
     // Increment kill counter
     GVAR(killCount) = GVAR(killCount) + 1;
-    GVAR(eventsArray) pushBack format ["Killed: %1 %2", _name, _killInfo];
-    GVAR(outputText) = (format ["Total Kills: %1<br/>", GVAR(killCount)]) + (GVAR(eventsArray) joinString "<br/>");
+    GVAR(eventsArray) pushBack format [LLSTRING(Killed), _name, _killInfo];
```

Comment:
looks like `<br/>` should be non-localized here and below

Reply:
I did it as it is at ACE3
https://github.com/acemod/ACE3/blob/3b15014099dc47140d0b9c8e925bab0c7a588c8a/addons/cargo/stringtable.xml#L146-L150

Reply:
yeah, looks like there too